src/components/Board.tsx

```diff
@@ -410,9 +423,10 @@ function Board() {
   };
 
   const [storyRefs, storyRects] = useMultipleRects({ ids: sprintStoryIds });
-
+  const currentBoardId = id;
   const boardApi = useMemo(
     () => ({
+      currentBoardId,
```

Comment:
Alternatively, you can just `curretBoardId: id` here instead of defining this above. Not a deal breaker tho

Reply:
Changed as per recommended in recent commit.